docs/docs/pricing/README.md

```diff
@@ -11,15 +11,238 @@ To support these goals, **Pipedream offers a [generous free tier](#developer-tie
 
 Read more about our plans and pricing options below.
 
-[[toc]]
-
-## Developer Tier
-
-**The Developer tier is free**. You can run any number of sources and workflows, as long as they operate under [Pipedream platform limits](/limits/).
-
-**To run any number of invocations, for any amount of time, [upgrade to the Professional tier](https://pipedream.com/pricing)**.
-
-### Developer Tier Support options
+## For Individuals
```

Comment:
Can you point people to the new pricing page for this table so the information is in one place? Also easier for them to purchase directly from that page.